src/test/java/uk/gov/hmcts/dts/fact/services/validation/ValidateServiceTest.java

```diff
@@ -29,6 +31,35 @@ class ValidateServiceTest {
     @Autowired
     private ValidationService validationService;
 
+    private static final String BAD_EPIM_ID = "BAD EPIM!";
+    private static final String GOOD_EPIM_ID = "GOOD-EPIM";
+    private static final CourtAddress BAD_EPIM_ADDRESS = new CourtAddress(
+        1,
+        null,
```

Comment:
Does it accept nulls? Interesting

Reply:
this is just a unit test that looks at the epim id the rest are null because i don't need any of those values

Reply:
I don't think we should pass in nulls if thats the case. You can put in dummy values instead. Reason being is that if the code changes in the future and utilises one of the other fields, we will see null pointer errors. Always try and avoid passing in nulls if possible and protect against them